app/models/king.rb

```diff
@@ -14,4 +14,37 @@ def valid_move?(x, y)
   def radial_move?(x, y)
     x_distance(x) <= 1 && y_distance(y) <= 1
   end
+
+  # Returns true if the king and rook have not moved,
+  # the location is a valid castling target, and there is
+  # no obstruction along the way. DOES NOT INCLUDE CHECK LOGIC.
+  def clear_castling_move?(x, y)
```

Comment:
This conforms with my clear_ modifier that modified all the original moveset validations to show that it was also checking for obstruction.  So if we remove clear here, we should remove it everywhere.  Clear essentially means = unobstructed, as I've used it.  Technically a castling move HAS to be unobstructed so you could argue clear could be dropped here, but for the other helper methods it cannot be so similarly implied because they are helper methods and not directly implementing a known chess rule.